ush/forecast_postdet.sh

```diff
@@ -883,7 +883,7 @@ CICE_postdet() {
   dumpfreq=${dumpfreq:-"y"} #  "h","d","m" or "y" for restarts at intervals of "hours", "days", "months" or "years"
 
   if [[ "${RUN}" =~ "gdas" ]]; then
-    cice_hist_avg=".false."   # DA needs instantaneous
+    cice_hist_avg=".false., .false., .false., .false., .false."   # DA needs instantaneous
   else
     cice_hist_avg=".true."    # P8 wants averaged over histfreq_n
```

Comment:
Should this be
```suggestion
    cice_hist_avg=".true., .true., .true., .true., .true."    # P8 wants averaged over histfreq_n
```

Reply:
It defaults to true, so this isn't strictly necessary.

Reply:
Up to you guys @WalterKolczynski-NOAA . The default is .true.